.snippets/text/products/reference/connect/overview/connect-timeline.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "title": "[Get Started with Connect]()",
+        "content": "Configure and use the Connect UI widget to easily add an intuitive, multichain asset transfer UI to your web applications.",
+        "icon": ":fontawesome-solid-1:"
+    },
+    {
+        "title": "[Add Custom Tokens to Connect ](Todo)",
+        "content": "Update your Connect deployment to support custom tokens and use a token allowlist.",
+        "icon": ":fontawesome-solid-2:"
+    },
+    { 
+        "title": "[Customize your Connect Widget](Todo)",
```

```suggestion
        "title": "[Customize Your Connect Widget](Todo)",
```

We follow chicago-style for title capitalization. This is a handy tool I use: https://titlecaseconverter.com/